### PR TITLE
Enabling CORS for websockets

### PR DESCRIPTION
**Pull request by geobeau**

### CODE REVIEW

**Comment by InformatiQ** [comment]
LGTM 
is this a bets practice to have a *?

**Comment by geobeau** [comment]
Not really. Maybe we can had an attributes in the configuration to change it ?

We could for example: 

- In preprod, we leave "*" to able to test it from anywhere
- in prod we could just whitelist domains where the portal will be

**Comment by geobeau** [comment]
For now, "*" will do but we should open a ticket to fix it later.

File: tailon/server.py [hunk 1]
```diff
@@ -47,7 +47,8 @@ class Files(BaseHandler):
     def get(self, check=None):
         self.application.file_lister.refresh()
         self.set_header('Cache-Control', 'no-store, no-cache, must-revalidate, max-age=0')
-        self.set_header('Content-Type', 'application/json')
+        self.set_header('Content-Type', 'application/json')  
+        self.set_header('Access-Control-Allow-Origin', '*')
```
Comment (iksaif) [inline tailon/server.py:51]: instead I'd just add a way to specify a set of custom headers in the configuration.

**Comment by iksaif** [comment]
:+1: let's merge that and clean later

**Pull request merged**